### DISCUSSION TITLE
Vocal Extraction - Pad Sound Effects Removal

### DISCUSSION THREAD

**Discussion by Tackoppe** [Anjok07/ultimatevocalremovergui#781] [Q&A]
Is there any tool to remove pad sound effects? I extracted vocals and it's pretty good, it's just surrounded by pad sound effects. 

I used:

1. Ensemble Mode
2. Vocals/Instrumental
3. Vocals Only
4. Max Spec/Min Spec
5. Exported to WAV format

Models 

1. VR Arc: 3_HP-Vocal-UVR
2. MDX-Net: UVR-MDX-NET Inst HQ 1
3. Demucs: v4 | htdemucs

**Comment by Julayla** [comment]
Yeah, I'm having trouble with this too. I mean I want to listen to the whole music of the What the Heck level from the Earthworm Jim Special Edition and there's that one woman scream still left in there and like the one above, I use most of what was used on there, including the UVR-MDX-NET Inst HQ 4 version. I need help too.
